README.md

```diff
@@ -281,7 +287,211 @@ Examples of WordPress block markup and the associated data structure returned by
 </tr>
 </table>
 
-## Preact example
+### GraphQL
+
+The GraphQL API, requires some setup before it can be it can be used.
+
+#### Setup
+
+The Block Data API integrates with **WP-GraphQL** to provide a GraphQL API as well. As a result, it is necessary to have WP-GraphQL installed, to activate the GraphQL API. To do so, follow the instructions mentioned [here](https://www.wpgraphql.com/docs/quick-start#install). 
```

Comment:
There isn't a separate step to activate the GraphQL endpoint, it's just plugin activation. Also linking the word "here" is a pet peeve (and bad for SEO). :)

```suggestion
The Block Data API integrates with **WPGraphQL** to provide a GraphQL API. It is necessary to have WPGraphQL [installed and activated](https://www.wpgraphql.com/docs/quick-start#install).
```

Reply:
Agree, addressed in https://github.com/Automattic/vip-block-data-api/pull/51/commits/fac6ebf59e071e5db10234cabe951f7db1d4894f!